SPECS/systemd/systemd.spec

```diff
@@ -241,6 +243,8 @@ fi
 %dir %{_includedir}/systemd
 /lib/libudev.so
 /lib/libsystemd.so
+/lib/systemd/libsystemd-core-254.so
```

Comment:
Could we use a version macro here to reduce manual updates when we upgrade between major systemd versions?

Reply:
sure, will update